Composer/packages/client/src/components/BotRuntimeController/ErrorCallout.tsx

```diff
@@ -81,6 +78,8 @@ fieldsWhiteList.set('code', { visible: false, name: 'code' });
 fieldsWhiteList.set('signal', { visible: false, name: 'signal' });
 fieldsWhiteList.set('stderr', { visible: false, name: 'stderr' });
 
+const DEFAULT_FONT_SETTINGS = getDefaultFontSettings();
```

Comment:
why is this using the "default" font settings instead font settings?

Reply:
Removed it. Its using the font of the parent.